docs/en/latest/plugins/openid-connect.md

```diff
@@ -275,6 +275,21 @@ upstream sent too big header while reading response header from upstream
 
 If so, try adjusting `proxy_buffers`, `proxy_buffer_size`, and `proxy_busy_buffers_size` to larger values.
 
+Another option is to set the `session_content` attribute:
+
+```yaml
+openid-connect:
+  ...
+  bearer_only: false
+  # this comes from https://github.com/zmartzone/lua-resty-openidc, if this is missing, all information will be stored in the session. This
+  # causes the cookies to be too large and the request to fail. We have control over id_token, user, enc_id_token, and access_token.
+  # We currently include only access_token in the session. If this gets too big we can remove it as well.
+  session_contents: 
+    access_token: true
```

Comment:
Technically this is an attribute that was previously not explicitly exposed, so should be also be added to attribute section of the doc; as well as to the plugin schema in the source code (relevant tests need to be updated as well) 💭 cc: @juzhiyuan